dbms/src/Formats/ProtobufWriter.cpp

```diff
@@ -1,3 +1,6 @@
+#include <Common/config.h>
+#if USE_PROTOBUF
```

Comment:
There are also files ProtobufSimpleWriter.h, ProtobufSimpleWriter.cpp.
Maybe their contents should be enclosed in `#if USE_PROTOBUF` .. `#endif` too, what do you think?